API/Controllers/ActivityController.cs

```diff
@@ -27,17 +29,26 @@ public async Task<IActionResult> CreateActivity(Activity activity)
             return Ok(await Mediator.Send(new Create.Command { Activity = activity }));
         }
 
+        [Authorize(Policy = "IsActivityHost")]
         [HttpPut("{id}")]
         public async Task<IActionResult> EditActivity (Guid id, Activity activity)
         {
             activity.Id = id;
             return Ok(await Mediator.Send(new Edit.Command { Activity = activity }));
         }
 
+        [Authorize(Policy = "IsActivityHost")]
         [HttpDelete("{id}")]
         public async Task<IActionResult> DeleteActivity (Guid id)
         {
             return Ok(await Mediator.Send(new Delete.Command { Id = id }));
         }
+
+        [HttpPost("{id}/attend")]
```

Comment:
Having verbs in the url path does not respect REST principles. So if we want to work with the attendee resource, we should in this case Patch the attendee resource. 

Something like: `PATCH api/attendees/{id}`

Reply:
Or if this is a modificaition on the activity resource, Patch the activity

Reply:
I changed it.